### ISSUE TITLE
There is no the Developer mode setting

### ISSUE THREAD

**Issue by vasil-sokolov** [toggl/track-extension#2245]
* **OS version**: Windows 11
* **Browser version**: Edge latest
* **Extension version**:  3.0.18

### 🐛 Describe the bug

In the current version of the extension, there is no Developer mode setting on the extension settings screen.

### Expected behaviour

Since the [Contributing ](https://github.com/toggl/track-extension/blob/master/docs/CONTRIBUTING.md ) article offers to enable this option, I expect it will be in place :)

**Comment by andresgz** [comment]
Right!, It used to be there, but It's not available anymore, so we can not help!

**Comment by willrowe** [comment]
See #2187 for some more info.

**Comment by d2718nis** [comment]
While the "Development mode" is missing from the extension itself and there's no source code left in this repo, it might be a good idea to remove any mentions of the said "Development mode" from the docs.

**Comment by matthewford** [comment]
or add the source code back into the repo so we can run our own extensions...
